java/src/jmri/jmrix/SystemConnectionMemo.java

```diff
@@ -28,6 +28,7 @@
     public static final String DISABLED = "ConnectionDisabled";
     public static final String USER_NAME = "ConnectionNameChanged";
     public static final String SYSTEM_PREFIX = "ConnectionPrefixChanged";
+    public static final String INTERVAL = "ConnectionIntervalChanged";
```

Comment:
Since INTERVAL stores the name of a property, the value of INTERVAL should be “outputInterval”, not “ConnectionIntervalChanged”. Since Java bean and introspection mechanisms can see a property outputInterval (it has a getter and setter), they should be able to subscribe to changes in that property, but firing change notifications for some other property (“ConnectionIntervalChanged”) prevents that.

Reply:
Change made